### PR TITLE
Add uninstall helm release action to list, detail and topology pages

### PR DESCRIPTION
**Pull request by rohitkrai03**
**Fixes**: https://issues.redhat.com/browse/ODC-3059

**Analysis / Root cause**: There is no way for a user to delete/uninstall a Helm release from UI.

**Solution Description**: Add action menu items for deleting helm release in helm release list page, helm release details page, context menu for helm release in topology and release details panel in topology.

**Screen shots / Gifs for design review**: 

Helm Delete

**Unit test coverage report**: 

**Test setup:**

The UI depends on APIs being added by https://github.com/openshift/console/pull/4580. Need to build the backend on top of that PR in order to test the API.

**Browser conformance**: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

cc: @openshift/team-ux-review

### CODE REVIEW

File: frontend/packages/dev-console/src/actions/modify-helm-release.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    label: 'Delete Helm Release',
+    callback: () => {
+      deleteResourceModal({
+        blocking: true,
+        resourceName: releaseName,
+        resourceType: 'Helm Release',
+        onSubmit: () => {
+          return coFetchJSON.delete(`/api/helm/release?name=${releaseName}&ns=${namespace}`);
+        },
+        close: redirect ? () => history.push(redirect) : undefined,
```
Comment (christianvogt) [inline frontend/packages/dev-console/src/actions/modify-helm-release.ts]: Redirect occurs even if the operation is cancelled. This should only redirect on successful submission.

File: frontend/packages/dev-console/src/actions/modify-helm-release.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+export const deleteHelmRelease = (releaseName: string, namespace: string, redirect?: string) => {
+  return {
+    label: 'Delete Helm Release',
+    callback: () => {
+      deleteResourceModal({
+        blocking: true,
+        resourceName: releaseName,
+        resourceType: 'Helm Release',
+        onSubmit: () => {
+          return coFetchJSON.delete(`/api/helm/release?name=${releaseName}&ns=${namespace}`);
```
Comment (christianvogt) [inline frontend/packages/dev-console/src/actions/modify-helm-release.ts]: What happens if the network call fails? Where does the error go?

File: frontend/packages/dev-console/src/actions/modify-helm-release.ts (hunk 1 above)
Comment (rohitkrai03) [reply frontend/packages/dev-console/src/actions/modify-helm-release.ts]: Fixed it.

File: frontend/packages/dev-console/src/actions/modify-helm-release.ts (hunk 2 above)
Comment (rohitkrai03) [reply frontend/packages/dev-console/src/actions/modify-helm-release.ts]: Updated the modal to show the error inline. Earlier it was showing the error but was also closing the modal immediately after.

**Pull request merged**